clang/include/clang/Driver/Action.h

```diff
@@ -629,11 +629,20 @@ class OffloadUnbundlingJobAction final : public JobAction {
           DependentOffloadKind(DependentOffloadKind) {}
   };
 
+  /// Allow for a complete override of the target to unbundle.
+  /// This is used for specific unbundles used for SYCL AOT when generating full
```

Comment:
What are specific unbundles?

Reply:
As this is part of the unbundling job class, the specific unbundled are bundles that were generated using the `-ftarget-device-link` option.